ui/src/hosts/containers/HostsPage.js

```diff
@@ -116,7 +116,7 @@ export class HostsPage extends Component {
       <div className="page hosts-list-page">
         <PageHeader
           titleText="Host List"
-          optionsComponents={this.optionsComponents}
+          optionsComponents={this.optionsComponents()}
```

Comment:
could you make options components a getter so there's no need for an inline invocation in the render? i suspect that may be consistent with how it was done elsewhere